### DISCUSSION TITLE
How to add a frame with a background image

### DISCUSSION THREAD

**Discussion by maxwangwyf** [TomSchimansky/CustomTkinter#2576] [Q&A]
How to add a frame with a background image

**Comment by Akascape** [comment]
@maxwangwyf 
1) Add a ctk frame
2) Add a ctk label in that ctk frame, import the image in that ctk label
3) Place the ctk label, change width height of image if required
4) Add other widget in that ctk frame...

If you face background transparency issues with other widgets in front of the image, then check this post: #2234
